### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by taylornowotny**
Added design onboarding instructions to step 2

### CODE REVIEW

**Comment by taylornowotny** [comment]
I tried to add my name in there as a point of contact but I guess github profile links don't work in this? Suggestion?

**Comment by taylornowotny** [comment]
> Yup, you need to use markdown syntax like this
> 
> ```
> 
> [JayaKrishnaNamburu](https://github.com/JayaKrishnaNamburu)
> ```
> 
> [JayaKrishnaNamburu](https://github.com/JayaKrishnaNamburu)

Thank you! Updated

File: docs/contributing.md [hunk 1]
```diff
@@ -6,9 +6,12 @@ If you want to contribute and don't know where to start, here is a step-by-step
 
 1. If you have an idea that is not already in [discussion](https://github.com/webstudio-is/webstudio/discussions) or [issues](https://github.com/webstudio-is/webstudio-builder/issues), please start a discussion or reach out on [discord](https://discord.gg/UNdyrDkq5r)
 
-1. We welcome everyone, developers and non-developers. If you are a developer and would like to build it - please say so and we will assign the issue to you, so you will have a complete ownership over it's development.
+1. We welcome everyone, developers and non-developers.
 
-   If you are a designer, you can lead the process too. We will find a developer who will pair up with you.
+   - If you are a DEVLOPER, and you want to build something with a UI, your first step is to become familiar with our design in figma. Start with the first page of our [Design Docs](https://www.figma.com/file/xCBegXEWxROLqA1Y31z2Xo/%F0%9F%93%96-Webstudio-Design-Docs?type=design&node-id=234%3A36754&t=w3VxT162RQF0gTrI-1). And for our design system & components see the [Webstudio Library](https://www.figma.com/file/sfCE7iLS0k25qCxiifQNLE/%F0%9F%93%9A-Webstudio-Library?type=design&node-id=2647%3A10046&t=f4xr8mcumXfXkHVh-1).
```
Comment (kof) [inline docs/contributing.md:11]: > to build something with a UI

to contribute to the Builder UI

Design in figma is all about builder

**Review by kof — APPROVED** [review]
one comment to fix, otherwise feel free to merge

**Pull request merged**